Tests/test_file_libtiff.py

```diff
@@ -686,12 +686,17 @@ def test_save_ycbcr(self, tmp_path: Path) -> None:
             assert reloaded.tag_v2[532] == (0, 255, 128, 255, 128, 255)
 
     def test_exif_ifd(self, tmp_path: Path) -> None:
```

Comment:
The `tmp_path` parameter can be removed now.

Reply:
I've updated the commit.